packages/sdk/src/oracle/store/has.ts

```
@@ -0,0 +1,21 @@
+import Read from "./read";
+import { ignoreError } from "../utils";
```

There's a helper function for ignoring errors?!?